webapp/src/main/java/org/opentestsystem/rdw/admin/model/CsvValidationResult.java

```diff
@@ -4,10 +4,11 @@
  * This model represents a CSV Validation failure containing the failing row
  * and failure message.
```

Comment:
This is why I don't like comments... I'll update....

Reply:
I'm with you. I like 'self documenting' code and comments when things are not clear because of this kind of maintenance difficulty as well.